### ISSUE TITLE
Provider: it's not possible to use the default `check-server` set for ipv4@ipv64.net to get an update of the `A` record.

### ISSUE THREAD

**Issue by hrzlgnm** [troglobit/inadyn#497]
The code in that the retrieves the ip https://github.com/troglobit/inadyn/blob/master/src/ddns.c#L139 to be used via the default set `ifconfig.me` defined in https://github.com/troglobit/inadyn/blob/master/plugins/ipv64.c#L53 is confused and assumes `ipv6` is enforced, due to the name `ipv64.net` containing `ipv6`.

This leads to an issue, that a definition like the following one:
```
provider ipv4@ipv64.net { 
    username = <REDACTED>
    password = none
    hostname = <REDACTED>
}
``` 
tries to update the ipv4 record using the retrieved ipv6 address.

I'm currently using the following workaround:
```
provider ipv4@ipv64.net { 
    ...
    checkip-command = "/usr/bin/curl -s -4 https://ifconfig.me/ip"
}
```

There are more constructs like the mentioned above.
https://github.com/troglobit/inadyn/blob/master/plugins/core-networks.c#L55
https://github.com/troglobit/inadyn/blob/master/plugins/porkbun.c#L228
https://github.com/troglobit/inadyn/blob/master/plugins/cloudflare.c#L252
https://github.com/troglobit/inadyn/blob/master/plugins/freedns.c#L72
https://github.com/troglobit/inadyn/blob/master/plugins/dnspod.c#L103
https://github.com/troglobit/inadyn/blob/master/plugins/dnspod.c#L172
https://github.com/troglobit/inadyn/blob/master/plugins/dnshome.c#L76
https://github.com/troglobit/inadyn/blob/master/src/ddns.c#L621

**Comment by hrzlgnm** [comment]
As proposed in #496

**Comment by hrzlgnm** [comment]
I'm also happy to tackle this, if i get some hints how to approach this properly.

**Comment by troglobit** [comment]
Unfortunately I have no time to guide anyone at the moment, but I'll do my best to review PRs (provided they are small and logical change sets).

**Comment by hrzlgnm** [comment]
Fair enough, I'll give it a shot with checking that the provider name starts with "ipv6" instead of containing it.
